src/ThermalScattering.tex

```diff
@@ -143,9 +163,39 @@ \subsubsection{\textsf{ITXE} Block}\label{sec:ITXEBlock}
   \label{tab:ITXEBlockContinuousData}
 \end{XSSTable}
 
-\subsubsection{\textsf{ITCA} Block}\label{sec:ITCABlock}
+\subsubsection{\textsf{ITCE} Block}\label{sec:ITCEBlock}\label{sec:ITCEIBlock}
```

Comment:
Why did this get moved below the ITXE Block description? I would think it makes most sense to place the block descriptions in the same order as given at the top of this section.

Reply:
Near as I can tell, it's the same as what was above, with the ITCEI starting information added.

Reply:
Because that is the order in which the blocks are actually given in the ACE files.